jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/ExceptionUtil.java

```diff
@@ -107,9 +109,31 @@ public static void ifExceptionThrowUnchecked(Throwable throwable)
             throw runtimeException;
         if (throwable instanceof Error error)
             throw error;
+        if (throwable instanceof IOException ioException)
+            throw new UncheckedIOException(ioException);
         throw new RuntimeException(throwable);
     }
 
+    /**
+     * Throw a {@link Throwable} as an unchecked {@link Exception}.
+     * @param cause The {@link Throwable} to that cause.
+     * @throws Error If the passed {@link Throwable} is an {@link Error}.
+     * @throws RuntimeException Otherwise, if the passed {@link Throwable} is not null.
+     */
+    public static void throwUnchecked(Throwable cause)
```

Comment:
Not returning the exception to throw makes much code awkward, like so:

```java
private boolean test()
```

Reply:
I would like to see this as `Throwable toUnchecked(Throwable x)` as well.

Reply:
@sbordet @joakime that doesn't work as throwing that at the calling site would not be unchecked.